crates/matrix-sdk/src/widget/mod.rs

```diff
@@ -139,7 +139,7 @@ impl WidgetDriver {
         let (incoming_msg_tx, mut incoming_msg_rx) = unbounded_channel();
 
         // Forward all of the incoming messages from the widget.
-        tokio::spawn({
+        matrix_sdk_common::executor::spawn({
```

Comment:
Could you use a `use` statement at the top, to avoid the fully-qualified path here and below, please?

Reply:
We do this in a lot of places. I was wondering if this is desired as it allows for easier ctrl+F searching.
But moving it to the import definitely sounds more idiomatic.